pkgs/development/python-modules/python3-xcaplib/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k, python3-application, lxml
+, gevent, python3-eventlib }:
+
+buildPythonPackage rec {
+  pname = "python3-xcaplib";
+  version = "2.0.1";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-xcaplib";
+    rev = version;
+    sha256 = "sha256-tIHDkseFkLqrdXFPSemYfOMRRhNcqJxCgMSr2xkI/DA=";
+  };
+
+  propagatedBuildInputs = [ lxml gevent python3-eventlib python3-application ];
+
```

Comment:
Could we have a `pythonImportsCheck` please?

Reply:
Will do.